### PR TITLE
[ERMP] Hooks

### PR DESCRIPTION
**Pull request by lvrubleuska**

### CODE REVIEW

File: src/Components/HeaderWrapper/HeaderWrapper.js [hunk 1]
```diff
@@ -1,19 +1,37 @@
...
-let HeaderWrapper = () => {
+let HeaderWrapper = ({ mainPage, detailMovie, onSearchIconClick }) => {
     return (
         <ErrorBoundary>
-            <div className="headerWrapper">
-                <Header />
-                <SectionSearch />
+            <div className={`headerWrapper ${mainPage ? "" : "detailPage"}`}>
+                {
+                    mainPage ?
```
Comment (AlexKarmaz) [inline src/Components/HeaderWrapper/HeaderWrapper.js:15]: mainPage seems like redundant flag. You can just use detailMovie.

**Review by AlexKarmaz — APPROVED** [review]
Leave several comments/recommendations. 
The rest looks OK to me. All requirements are met.

File: src/Components/App/App.js [hunk 2]
```diff
@@ -1,14 +1,28 @@
...
 import HeaderWrapper from "../HeaderWrapper";
 import SectionMovies from "../SectionMovies";
 import Footer from "../Footer";
 import "./App.css";
 
 let App = () => {
+    let [mainPage, setMainPage] = useState(true);
+    let [detailMovie, setDetailMovie] = useState({});
+
+    let changePage = (bMainPage, oMovie) => {
```
[the remark this answers is not in the document]
Comment (lvrubleuska) [reply src/Components/App/App.js:11]: This is the Hungarian notation where name prefixes indicate the type of variables. b stands for boolean and o for object here.

File: src/Components/HeaderWrapper/HeaderWrapper.js (hunk 1 above)
Comment (lvrubleuska) [reply src/Components/HeaderWrapper/HeaderWrapper.js]: Nice point, fixed. Thank you!

**Pull request merged**